### PR TITLE
Release 0.14

### PR DESCRIPTION
**Pull request by deadprogram**
Pretty much all the changes needed for the 0.14.0 release.

@aykevl or @jaddr2line we might also want to review some of the pages under https://github.com/tinygo-org/tinygo-site/tree/dev/content/compiler-internals

### CODE REVIEW

File: content/lang-support/stdlib.md [hunk 1]
```diff
@@ -136,35 +137,36 @@ sync |  <span style="color: green">✔</span> yes  |
 sync/atomic |  <span style="color: green">✔</span> yes  | 
 syscall |  <span style="color: green">✔</span> yes  | 
 syscall/js |  <span style="color: green">✔</span> yes  | 
-testing |  [<span style="color: red">✗</span> no](#testing)  | 
-testing/iotest |  <span style="color: green">✔</span> yes  | 
+testing |  <span style="color: green">✔</span> yes  | 
+testing/iotest |  [<span style="color: red">✗</span> no](#testing-iotest)  |
```
Comment (aykevl) [inline content/lang-support/stdlib.md:141]: (Not commenting on this PR but rather on the TinyGo 0.14 release)

That looks like a regression, but probably not important.

File: content/lang-support/stdlib.md [hunk 2]
```diff
@@ -136,35 +137,36 @@ sync |  <span style="color: green">✔</span> yes  |
...
 text/tabwriter |  <span style="color: green">✔</span> yes  | 
 text/template |  [<span style="color: red">✗</span> no](#text-template)  | 
-text/template/parse |  [<span style="color: red">✗</span> no](#text-template-parse)  | 
+text/template/parse |  <span style="color: green">✔</span> yes  | 
 time |  <span style="color: green">✔</span> yes  | 
 unicode |  <span style="color: green">✔</span> yes  | 
 unicode/utf16 |  <span style="color: green">✔</span> yes  | 
 unicode/utf8 |  <span style="color: green">✔</span> yes  | 
 unsafe |  <span style="color: green">✔</span> yes  | 
-vendor/golang.org/x/crypto/chacha20poly1305 |  <span style="color: green">✔</span> yes  |
```
Comment (aykevl) [inline content/lang-support/stdlib.md:151]: I would guess these errors are caused by the switch to Go modules.

    use of vendored package not allowed

I think they can be ignored, none of the existing crypto related packages have stopped compiling. And you shouldn't import these vendor directories anyway.

File: content/microcontrollers/drivers/_index.md [hunk 3]
```diff
@@ -24,13 +24,15 @@ The following 48 devices are supported.
 | [BlinkM RGB LED](http://thingm.com/fileadmin/thingm/downloads/BlinkM_datasheet.pdf) | I2C |
 | [BME280 humidity/pressure sensor](https://cdn-shop.adafruit.com/datasheets/BST-BME280_DS001-10.pdf) | I2C |
```
Comment (aykevl) [inline content/microcontrollers/drivers/_index.md:25]: I think you're missing the BMI160 here.

**Review by aykevl — COMMENTED** [review]
I didn't check the board documentation (I assume it's correct). I did check other changes and have one comment. Other than that, this looks fine.

Yeah, there are probably things that need updating in the compiler internals, I'll need to check that.

File: content/microcontrollers/feather-nrf52840.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    ```
+
+- The Adafruit Feather nRF52840 board should restart and then begin running your program.
+
+### Troubleshooting
+
+If you have troubles getting your Adafruit Feather nRF52840 board to receive code, try this:
+
+- Press the "RESET" button on the board two times to get the Adafruit Feather nRF52840 board ready to receive code.
+- The CLUE board will appear to your computer like a USB drive.
```
Comment (sago35) [inline content/microcontrollers/feather-nrf52840.md]: I think you forgot to change it to `Feather nRF52840 board`.

Comment (deadprogram) [reply content/microcontrollers/feather-nrf52840.md]: Changed.

File: content/microcontrollers/drivers/_index.md (hunk 3 above)
Comment (deadprogram) [reply content/microcontrollers/drivers/_index.md:25]: Added.

File: content/lang-support/stdlib.md (hunk 1 above)
Comment (deadprogram) [reply content/lang-support/stdlib.md:141]: Agreed, or might be is correctly failing import now?

File: content/lang-support/stdlib.md (hunk 2 above)
Comment (deadprogram) [reply content/lang-support/stdlib.md:151]: I added some code to ignore vendored directories, and regenerated the imports report.

Is this good now, @aykevl ?

**Comment by aykevl** [comment]
Yes looks good now.

**Comment by deadprogram** [comment]
OK, cool. Now merging into `dev`.

**Pull request merged**